examples/simple_neo4j.py

```diff
@@ -0,0 +1,67 @@
+import sycamore
+from sycamore.transforms.extract_document_structure import StructureByDocument
+from sycamore.transforms.extract_graph_entities import EntityExtractor
+from sycamore.transforms.extract_graph_relationships import RelationshipExtractor
+from sycamore.transforms.partition import ArynPartitioner
+
+from sycamore.llms import OpenAI, OpenAIModels
+
+from typing import Optional
+from pydantic import BaseModel, Field
+
+llm = OpenAI(OpenAIModels.GPT_4O_STRUCTURED)
+ctx = sycamore.init()
+
+
+class Report(BaseModel):
+    ID: str = Field(
+        description="This ID can be found under either Accident Number, Incident Number, or Occurance Number"
+    )
+
+
+class Aircraft(BaseModel):
+    registration: str
+    make: str
+    model: str
+
+
+class Pilot(BaseModel):
+    is_student: bool
+    total_flight_hours: Optional[int]
+
+
+class INVOLVED_AIRCRAFT(BaseModel):
+    start: Report
+    end: Aircraft
+
+
+class FLOWN_BY(BaseModel):
```

Comment:
why the UPPERCASE NAMES?

Reply:
Default way neo4j does it, not required but thought it would just be consistent